webrtc-stats.html

```diff
@@ -3654,9 +3658,10 @@ <h2>
                   might be higher.
                 </p>
                 <p>
-                  Only [= map/exist =] when a receiver-side bandwidth estimate was calculcated.
-                  Does not [= map/exist =] for candidate pairs that were never used for receiving
-                  packets that were taken into account for bandwidth estimation or
+                  Only [= map/exist =]s when a receive-side bandwidth estimation such as
```

Comment:
Rethinking about this, a receiving endpoint does not necessarily need REMB to measure the incoming bitrate, it can calculate based on the packets received, bytes received, and even incorporate packet losses.

Reply:
It can, in addition to calculating a delay-based estimate. Not done by any implementation I am aware of though.